lib/tasks/remove_image_alt_text.rake

```diff
@@ -0,0 +1,23 @@
+require 'csv'
+
+namespace :news_article do
+  desc "Remove inaccessible alt text"
+  task remove_inaccessible_alt_text: :environment do
+
+    ids = CSV.read(File.join(Rails.root, 'lib', 'tasks', 'newsarticle_alt_text_ids.csv'), :headers=>true)
```

Comment:
```suggestion
    ids = CSV.read(Rails.root.join('lib', 'tasks', 'newsarticle_alt_text_ids.csv'), :headers=>true)
```

I think the linter might complain if you don't do this.

Reply:
seem to have got away with it, but will change it as it's much nicer, thanks!